### PR TITLE
Feature/improve mouse input

### PR DESCRIPTION
**Pull request by adreyfus**

### CODE REVIEW

File: src/inputSources/mouseInput.js [hunk 1]
```diff
@@ -89,6 +89,12 @@ function mouseDown (e) {
     type: eventType
   };
 
+  const onMouseDown = $(element).data('onMouseDown');
```
Comment (swederik) [inline src/inputSources/mouseInput.js]: This seems like a weird way to add a function to run on an element on mouse down. What are you using this for? Why shouldn't users just add an event handler to CornerstoneToolsMouseDown instead?

**Comment by swederik** [comment]
Thanks for your contribution!

**Comment by adreyfus** [comment]
You're absolutely right, I did not see that the event 'CornerstoneToolsMouseDown' was triggered on the viewportElement. Sorry :)

**Comment by swederik** [comment]
Thanks!

**Pull request merged**